samcli/local/init/templates/cookiecutter-aws-sam-hello-ruby/{{cookiecutter.project_name}}/README.md

```diff
@@ -31,27 +32,16 @@ Setup Ruby Version Manager from [Ruby Version Manager](http://rvm.io/)
 Run following commands
 
 ```bash
-rvm install ruby-2.5.0
-rvm use ruby-2.5.0
-rvm --default use 2.5.0
+rvm install ruby-2.5.3
```

Comment:
It's a best practice, yes. Ruby on AWS Lambda uses version 2.5.3 - while 2.5.0 will technically work for builds, there are security patches present on 2.5.3 that are not present on 2.5.0, so I would not intentionally direct users to use an older version in this case.

Reply:
Is there a way the newest just gets installed. I am worried this will be out of date and something we need to maintain.

Reply:
do we need to provide instructions on how to use install Ruby for Ruby developers?  I wonder if Requirements (ruby 2.5) suffices as we do with Python and Node init

Reply:
This is how you install Ruby. Not sure what the question is @heitorlessa

Reply:
Sure, let me try once more and let me know if it's clearer: 

1. Do we need to tell Ruby developers how they should install Ruby?
2. Customers trying Ruby quickstart should have Ruby installed, shouldn't we assume that and cut the Installation part?

Reply:
I wouldn't add install instructions. People who use languages like ruby, go etc will have their tooling in place.

It would be like every pkg internally saying start off by installing the Amazon builder tools.